eng/devices/android.cake

```diff
@@ -473,8 +475,42 @@ void CleanUpVirtualDevice(AndroidEmulatorProcess emulatorProcess, AndroidAvdMana
 
 	// kill the process if it has not already exited
 	Information("emulatorProcess.Kill()");
-	try { emulatorProcess.Kill(); }
-	catch { }
+	try 
+	{ 
+		emulatorProcess.Kill();
```

Comment:
Go back to when you were wrapping this with a timeout. This operation itself is hanging for hours at a time and we want to make it so we just kill the call ot this if it times out